lib/src/over_react_redux/devtools/middleware.dart

```diff
@@ -42,11 +42,12 @@ class _OverReactReduxDevToolsMiddleware extends MiddlewareClass {
   final Logger log = Logger('OverReactReduxDevToolsMiddleware');
 
   _OverReactReduxDevToolsMiddleware() {
+    var windowConsole = getProperty(window, 'console');
     log.onRecord.listen((rec) {
       if (rec.level == Level.WARNING) {
-        window.console.warn('${log.name} [${rec.level.name}]: ${rec.message}');
+        callMethod(windowConsole, 'warn', ['${log.name} [${rec.level.name}]: ${rec.message}', if (rec.error != null) rec.error]);
```

Comment:
changed to this to allow for "grouping" the error with the warning using console.warns varargs